app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -595,13 +616,13 @@ class CampaignStacExport(
             _
           )
           ) =>
-        Some((name, ingestLocation))
+        Some(ingestLocation)
       case _ => None
     }
     val signedUrlDurationInDays = 7
     for {
-      maybeSignedURLAsset: Option[Tuple2[String, StacAsset]] <- maybeNameAndIngestLocation match {
-        case Some((name, ingestLocation))
+      maybeSignedURLAsset: Option[Tuple2[String, StacAsset]] <- maybeIngestLocation match {
```

Comment:
And this can be a nice `traverse` 🎉 (really should have been a `traverse` to begin with -- I apologize for the actions of past me, he was foolish)

Reply:
Wound up skipping this one since I immediately broke the types when I did it and the type error was a big ol mess